string object
string feature
geometry_msgs/Point point
